docs/components/sidebarIndex.js

```diff
@@ -17,9 +17,10 @@ const sidebarIndex: Array<sidebarIndexType> = [
   {
     sectionName: 'Getting Started',
     pages: [
-      "What's new",
+      'Roadmap',
+      "What's New",
```

Comment:
These look like regressions from the merge conflicts

Reply:
Ryan, you are incredible. Thank you so much.